statistics/histogram.go

```diff
@@ -281,6 +281,35 @@ func (hg *Histogram) BucketToString(bktID, idxCols int) string {
 	return fmt.Sprintf("num: %d lower_bound: %s upper_bound: %s repeats: %d ndv: %d", hg.bucketCount(bktID), lowerVal, upperVal, hg.Buckets[bktID].Repeat, hg.Buckets[bktID].NDV)
 }
 
+// BinarySearchRemoveVal removes the value from the TopN using binary search.
+func (hg *Histogram) BinarySearchRemoveVal(valCntPairs TopNMeta) {
```

Comment:
```BinarySearchRemoveVal``` is the same as the ```RemoveVal```.

Reply:
There have been several other methods to find a value in the Histogram, it may be better to try to reuse them, like `hg.locateBucket()` and `hg.Bounds.LowerBound()`.

Reply:
I find this problem too. but because this PR need to cherry-pick for old PR. we need to make changement simple. I will improve this code next next pr.